### PR TITLE
results table and timeline new UI

### PR DESCRIPTION
**Pull request by sulimani55**

### CODE REVIEW

File: src/app/@ansyn/ansyn/ansyn/ansyn.component.html [hunk 1]
```diff
@@ -14,6 +14,14 @@
 			<ansyn-context-menu *ngIf="renderContextMenu"></ansyn-context-menu>
 		</div>
 
+		<button [class.active]="toggleResults" class="results-table" (click)="toggleResultsTable()">
```
Comment (tltk90) [inline src/app/@ansyn/ansyn/ansyn/ansyn.component.html:17]: it is not better to enter this button inside the result-table component?

File: src/app/@ansyn/ansyn/ansyn/ansyn.component.less [hunk 2]
```diff
@@ -20,6 +20,55 @@
 		}
 	}
 
+	.results {
```
Comment (tltk90) [inline src/app/@ansyn/ansyn/ansyn/ansyn.component.less:23]: same as above

**Review by tltk90 — CHANGES_REQUESTED** [review]
in general i think all the thing according to result-table should be inside the
`result-table` component.

File: src/app/@ansyn/ansyn/modules/menu-items/results/components/results-table/results-table.component.less [hunk 3]
```diff
@@ -174,55 +185,15 @@
...
 
 					.results-table-auto-save-col {
 						padding-top: 20px;
-					}
-				}
-			}
-		}
-	}
-
-	&.rtl {
```
Comment (ramtob) [inline src/app/@ansyn/ansyn/modules/menu-items/results/components/results-table/results-table.component.less:191]: RTL code missing? Please add it.

**Please check the new design in both LTR and RTL modes** 
(they are switched according to "direction" field in translations in config)

File: src/app/@ansyn/ansyn/modules/menu-items/results/components/results/results.component.less [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+:host {
+	bottom: 0;
+
+	.results {
+		top: 0;
+		right: 0;
+		position: absolute;
+	}
+
+	button {
```
Comment (ramtob) [inline src/app/@ansyn/ansyn/modules/menu-items/results/components/results/results.component.less:12]: This definition should not be here. There is no `button` in the HTML

File: src/app/@ansyn/ansyn/modules/menu-items/results/components/results/results.component.ts [hunk 5]
```diff
@@ -1,16 +1,29 @@
...
+export class ResultsComponent {
 
 	constructor() {
 	}
 
-	ngOnInit() {
+	onExpandStart() {
+		const resultsTableElement = document.querySelector('.results');
+		if (resultsTableElement) {
+			resultsTableElement.setAttribute('style', `z-index: 5`);
```
Comment (ramtob) [inline src/app/@ansyn/ansyn/modules/menu-items/results/components/results/results.component.ts:26]: Why do you change `z-index` on expand?

File: src/app/@ansyn/ansyn/modules/overlays/components/overlay-overview/overlay-overview.component.ts [hunk 6]
```diff
@@ -107,10 +112,9 @@ export class OverlayOverviewComponent implements OnInit, OnDestroy {
 			if (!hoveredElement) {
 				return;
 			}
-			const hoveredElementBounds: ClientRect = hoveredElement.getBoundingClientRect();
-			this.myCurrentWidth = (this.el.nativeElement as HTMLElement).offsetWidth;
-			this.left = customElement ? this.isRTL ? hoveredElementBounds.left - this.myCurrentWidth : hoveredElementBounds.right : this.getLeftPosition(hoveredElementBounds.left);
-			this.top = hoveredElementBounds.top + (customElement ? hoveredElementBounds.height : 0);
+			const { left, height, top }: ClientRect = hoveredElement.getBoundingClientRect();
+			this.left = left - (customElement ? 275 : 150);
```
Comment (ramtob) [inline src/app/@ansyn/ansyn/modules/overlays/components/overlay-overview/overlay-overview.component.ts:116]: Why this change?
Please don't use explicit numbers, but calculations, as in master branch.

File: src/app/@ansyn/ansyn/modules/status-bar/components/search-panel/search-panel.component.html [hunk 7]
```diff
@@ -21,45 +21,7 @@
...
-	<ansyn-timepicker-presets *ngIf="isActive('TimePickerPreset')"
-							  (hideMe)="toggleExpander('TimePickerPreset')"
-							  (openTimePicker)="toggleExpander('TimePicker')"></ansyn-timepicker-presets>
-
-	<ansyn-timepicker *ngIf="isActive('TimePicker')"
-					  [timeRange]="timeRange"
-					  (closeTimePicker)="toggleExpander('TimePicker')"
-	></ansyn-timepicker>
-</div>
+<ansyn-time-picker-container [openTop]="false"></ansyn-time-picker-container>
```
Comment (ramtob) [inline src/app/@ansyn/ansyn/modules/status-bar/components/search-panel/search-panel.component.html:24]: 👍

File: src/app/@ansyn/ansyn/modules/status-bar/components/timepicker-presets/timepicker-presets.component.less [hunk 8]
```diff
@@ -10,6 +10,12 @@
 		top: @search-panel-popup-top;
 		min-width: 150px;
 		min-height: 110px;
+		margin-bottom: 9px !important;
```
Comment (ramtob) [inline src/app/@ansyn/ansyn/modules/status-bar/components/timepicker-presets/timepicker-presets.component.less:13]: Please don't use `important` (considered bad practice)

File: src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.ts [hunk 9]
```diff
@@ -159,3 +159,4 @@ export class AnnotationContextMenuComponent implements OnInit, OnDestroy {
 		return '';
 	}
 }
+
```
Comment (ramtob) [inline src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.ts:162]: 👍

**Review by ramtob — CHANGES_REQUESTED** [review]
Nice work

**Review by ramtob — APPROVED** [review]
👍

**Review by tltk90 — APPROVED** [review]

**Pull request merged**